core/src/main/java/com/linecorp/armeria/client/HttpClientFactory.java

```diff
@@ -166,6 +168,8 @@ final class HttpClientFactory implements ClientFactory {
         this.options = options;
 
         clientDelegate = new HttpClientDelegate(this, addressResolverGroup);
+
+        NettyMetricSupport.setup(meterRegistry, new MeterIdPrefix("armeria.event.loop.group"), workerGroup);
```

Comment:
- How about just `armeria.event.loops`?
- Should we distinguish client and server stats from each other by adding some tags?

Reply:
> Should we distinguish client and server stats from each other by adding some tags?

Yea, in theory this could be useful. This isn't something we did in Finagle (we used global counter) as in practice, there was just one event-loop-group per process at Twitter. 

We can probably add a tag (those are labels, right?) for a client / server if you feel strongly. What do you think?